app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> { order(created_at: :desc) }
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
+  validates :content, presence: true, length: { maximum: Settings.validates_content }
+  validate  :picture_size
```

Comment:
```suggestion
  validate :picture_size
```
có bao nhiêu loại validate tất cả?

Reply:
Chị hỏi về có bao nhiêu loại validate em sử dụng ạ, hay có bao nhiêu loại validate ạ